src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -6,8 +6,29 @@
     <IncludeInPackage>Microsoft.ML.AutoML</IncludeInPackage>
     <PackageDescription>ML.NET AutoML: Optimizes an ML pipeline for your dataset, by automatically locating the best feature engineering, model, and hyperparameters</PackageDescription>
     <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);CopyProjectReferencesToPackage</TargetsForTfmSpecificBuildOutput>
+
+    <!-- 
+      1591: Documentation warnings
+      NU5100: Warning that gets triggered because a .dll is not placed under lib folder on package. This is by design as we want MDAI to be under interactive-extensions folder.
+       -->
+    <NoWarn>$(NoWarn);1591;NU5100;MSML_GeneralName;MSML_ParameterLocalVarName;MSML_PrivateFieldName;MSML_TypeParamName;SA1028;SA1507;SX1101;MSML_NoInstanceInitializers</NoWarn>
```

Comment:
Do you need all these in the NoWarn? Or just the 1591 and NU5100? Having as few here as possible is better.

Reply:
I'm not sure I copied from here:

https://github.com/dotnet/machinelearning/blob/main/src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj#L15

Reply:
Embarrassingly I didn't really read beyond the comment. Let me see what I can remove.